spec/support/page_transition_support.rb

```diff
@@ -1,12 +1,16 @@
+# frozen_string_literal: true
+
 module PageTransitionSupport
-  def change_language(languageCode)
+  def change_language(language_code)
     within 'select[name=locale]' do
-      find("option[value=#{languageCode}]").click
-      expect(find("option[value=#{languageCode}][selected]")).to be_present
+      find("option[value=#{language_code}]").click
+      expect(
+        find("option[value=#{language_code}][selected]", wait: 5)
```

Comment:
The key part is `wait: 5`. The default wait time is 2 seconds, which might not be enough time for the page to reload.

All of the other changes are Rubocop fixes.

Reply:
If people still have problems, we can increase this further. There's not much downside to having a long wait time, because if it sees the element earlier, it won't wait the entire duration.

Reply:
Hmmm should we be relying on waits and sleeps to solve these flakey test problem?

Reply:
Sleeps, no. A sleep would pause execution for 5 seconds minimum. A wait in Capybara is more like a time out: it will wait _up to_ 5 seconds for the element to appear.

This is the best way that I know of to solve this kind of problem with Capybara.